bundle-core/src/main/java/net/discdd/utils/FileUtils.java

```diff
@@ -17,4 +31,56 @@ public static void createFileWithDefaultIfNeeded(Path path, byte[] bytes) throws
     public static void createEmptyFileIfNeeded(Path path) throws IOException {
         createFileWithDefaultIfNeeded(path, new byte[0]);
     }
+
+    public static String getFileNamesListFromDirectory(String directory) {
+        String fileList = "";
+        File dir = new File(directory);
+        File[] Files = dir.listFiles();
+        if (Files != null && Files.length > 0) {
+            for (File file : Files) {
+                fileList = fileList + "," + file.getName();
+            }
+        }
+        return fileList;
+    }
+
+    public static OutputStream getFilePath(BundleDownloadResponse response, String Receive_Directory) throws IOException {
+        String fileName = response.getMetadata().getBid();
+        File directoryReceive = new File(Receive_Directory + File.separator + response.getMetadata().getTransportId());
+        if (!directoryReceive.exists()) {
+            directoryReceive.mkdirs();
+        }
+
+        return Files.newOutputStream(Paths.get(
```

Comment:
use commas instead of separators

Reply:
why'd we want to use commas in file path?

Reply:
I think Paths.get method will place separators between parameters like Paths.get(dir1, dir2, ..., dir) will return dir1/dir2/.../dir